portfolio/src/main/java/com/google/sps/servlets/DataServlet.java

```diff
@@ -51,7 +51,14 @@ public String getName() {
       return this.name;
     }
   }
+
+  private static final String ENTITY_MESSAGE_PROPERTY_NAME = "message";
+  private static final String ENTITY_NAME_PROPERTY_NAME = "name";
+  private static final String ENTITY_NAME = "Comment";
   
+  /**
+   * Server side handler for POST requests.
```

Comment:
Method doc should begin with a verb phrase. 

```suggestion
   * Handler for server-side POST requests.
```